Dockerfile

```
@@ -36,7 +36,8 @@ RUN \
                    /usr/bin/ps \
                    && \
     if [[ $DVER == 8 ]]; then \
-        yum-config-manager --enable powertools; \
+        yum-config-manager --enable powertools && \
+        yum-config-makager --setopt=install_weak_deps=False --save > /dev/null; \
```

```suggestion
        yum-config-manager --setopt=install_weak_deps=False --save > /dev/null; \
```

Feel free to squash this fixup commit. Could you flesh out what problem this solves in the commit body while you're at it?

---

Done.